src/Data/UndefinedOr.purs

```diff
@@ -38,16 +38,27 @@
 -- | by clever use of typeclasses.
 -- |
 -- | You should especially Not use this over Maybe.
+-- |
+-- | Please also note that this (althoug it seems like a instance of) is not Functor
```

Comment:
Style remark: remove brackets and change the sentence: `s/that this (.*) is not Functor/that this is not a Functor, \1 it/`
Also typo in 'although'